assets/training/distillation/components/pipeline/spec.yaml

```diff
@@ -259,7 +259,7 @@ jobs:
 
   oss_chat_completion_finetune:
     type: command
-    component: azureml:oss_chat_completion_finetune:0.0.19
+    component: azureml:oss_chat_completion_finetune:0.0.21
```

Comment:
lets update the data import one as well post oss ft component release

Reply:
why are we jumping 2 versions?

Reply:
ayush: holding on here based on offline discussions
vikas: distillation component was running an older version already, we are using 19 but the latest is 21